.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,37 @@
+name: build-docs
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    if: ${{ startsWith(github.ref , 'refs/tags') != true && startsWith(github.ref , 'build-') != true }}
+    timeout-minutes: 90
+    name: Build Docs
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Print env'
+        run: |
+          echo "'uname -s' is:"
+          echo "uname: " $(uname)
+          echo "uname -m: " $(uname -m)
+          echo "uname -r:" $(uname -r)
+          echo "uname -s: " $(uname -s)
+          echo "uname -v: " $(uname -v)
+          printenv
+        shell: bash
+
+      - name: Install Doxygen (linux only)
+        run: |
+          set -e pipefail
+          # Install doxygen *before* running cmake
+          sudo apt-get install -y  doxygen
+        shell: bash
+        if: ${{ runner.os == 'Linux' }}
+
+      - name: Check formatting (linux only)'
```

Comment:
guessing 'name' (not 'run') bad for 'build-docs.yml' content? ("Build Docs (? deployment)" ?)

Reply:
Fixed